src/mam/mod_mam_pm.erl

```diff
@@ -215,13 +208,14 @@ disco_local_features(Acc, _, _) ->
 %%
 %% Note: for outgoing messages, the server MUST use the value of the 'to'
 %%       attribute as the target JID.
--spec user_send_packet(Acc :: mongoose_acc:t(), From :: jid:jid(),
-                       To :: jid:jid(),
-                       Packet :: exml:element()) -> mongoose_acc:t().
-user_send_packet(Acc, From, To, Packet) ->
+-spec user_send_packet(Acc, Args, Extra) -> {ok, Acc} when
+       Acc :: mongoose_acc:t(),
+       Args :: #{from := jid:jid(), to := jid:jid(), packet := exml:element()},
+       Extra :: map().
+user_send_packet(Acc, #{from := From, to := To, packet := Packet}, _) ->
```

Comment:
Wait, didn't we agree in https://github.com/esl/MongooseIM/pull/3769#discussion_r981035893 that we would extract the `from`, `to` and `packet` from the accumulator? 🤔
Same applies for the inbox PR I just merged now that I noticed. Not that it's critical because we'll rework this same `user_send_packet` hook heavily in the c2s rework 🤔

Reply:
Oh sorry, I forgot about it :/ I'll change it here and in the previous module.

Reply:
done 👍